pkg/extensions/tbtc/liquidation_recovery_test.go

```diff
@@ -0,0 +1,158 @@
+package tbtc
+
+import (
+	"bytes"
+	cecdsa "crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/keep-network/keep-ecdsa/pkg/ecdsa"
+)
+
+func Test_PublicKeyToP2WPKHScriptCode_Works(t *testing.T) {
```

Comment:
We've got just one test  for `PublicKeyToP2WPKHScriptCode` so we can drop `_Works`.
```suggestion
func TestPublicKeyToP2WPKHScriptCode(t *testing.T) {
```